File: C#/http-client.cs
using System;
using RestSharp;
namespace HelloWorldApplication {
  class HelloWorld {
    static void Main(string[] args) {
      var client = new RestClient("https://api.ipengine.dev");
      client.Timeout = -1;
      var request = new RestRequest(Method.GET);
      var body = @"";
      request.AddParameter("text/plain", body,  ParameterType.RequestBody);
      IRestResponse response = client.Execute(request);
      Console.WriteLine(response.Content);
    }
  }
}

File: JavaScript/http-client.js
const fetch = require("node-fetch");

var raw = ""

var requestOptions = {
    method: "GET",
    body: raw,
    redirect: "follow"
}

fetch("https://api.ipengine.dev", requestOptions)
    .then(response => response.text())
    .then(result => console.log(result))
    .catch(error => console.log("error", error))